bokeh/tile_providers.py

```diff
@@ -65,46 +115,48 @@
 
 '''
 
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 # Boilerplate
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 import logging
+
 log = logging.getLogger(__name__)
 
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 # Imports
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 
 # Standard library imports
 import sys
 import types
+from enum import Enum
```

Comment:
We still support Python 2 at least through the end of the year, so unfortunately we cannot use the new enum feature in core Python 3. For now we will have to use our hand-rolled `Enum` class in `bokeh.core.enums`:

https://github.com/bokeh/bokeh/blob/master/bokeh/core/enums.py

Reply:
I have not thought at all about how using our `Enum` might be different (hopefully not much) but let me know if you run into any questions/problems.